sktime/transformations/multiplexer.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements forecaster for selecting among different model classes."""
+
+from sklearn.base import clone
+
+from sktime.base import _HeterogenousMetaEstimator
+from sktime.transformations._delegate import _DelegatedTransformer
+from sktime.transformations.base import BaseTransformer
+
+__author__ = ["miraep8"]
+__all__ = ["MultiplexTransformer"]
+
+
+class MultiplexTransformer(_DelegatedTransformer, _HeterogenousMetaEstimator):
+    """Facilitate an AutoML based selection of the best transformer.
+
+    When used in combination with either TransformedTargetForecaster or
+    ForecastingPipeline in combination with ForecastingGridSearchCV MultiplexTransformer
+    provides a framework for transformer selection.  It can be used with exogenous
+    transformers (ie data to be applied to exogenous X variables) or transformers on
+    the y data, simply select the appropriate pipeline.
+
+    MultiplexTransformer has 2 parameters:
```

Comment:
seems redundant with the `Parameters` section?
Though, might be useful.

What would be important is to describe *behaviour*. I.e., what will the transformer do in `fit`, `transform`, `inverse_transform`?

Reply:
Agreed, I got rid of it and put more clarity on what tasks are delegated and how.